cypress/integration/26-register_registration/creator_navigates_to_resource_type_tab/navigate_to_resource_type.js

```diff
@@ -0,0 +1,272 @@
+  cy.get('[data-testid=button-next-tab]').click({ force: true });
+  cy.get('[data-testid=button-previous-tab]').click({ force: true });
+});
+And('they select the Resource type "Contribution to journal"', () => {
+  cy.get('[data-testid=publication-context-type]').click({ force: true }).type(' ');
+  cy.get('[data-testid=publication-context-type-Journal]').click({ force: true });
+  cy.wrap(JOURNAL_SUBTYPES).as('subTypes');
+  cy.wrap(JOURNAL_FIELDS).as('fields');
+});
+And('they select the Resource type "Book"', () => {
+  cy.get('[data-testid=publication-context-type]').click({ force: true }).type(' ');
+  cy.get('[data-testid=publication-context-type-Book]').click({ force: true });
+  cy.wrap(BOOK_SUBTYPES).as('subTypes');
+  cy.wrap(BOOK_FIELDS).as('fields');
+});
+Then('they see a Search box for "Publisher name"', () => {
+  cy.get('[data-testid=publisher-search-field]').should('be.visible');
+});
+And('they select the Resource Type', (dataTable) => {
+  cy.get('[data-testid=publication-context-type]').click({ force: true }).type(' ');
+  cy.get(`[data-testid=${RESOURCE_TYPES[dataTable.rawTable[0]]}]`).click({ force: true });
+  cy.wrap(CHAPTER_SUBTYPES).as('subTypes');
+  cy.wrap(CHAPTER_FIELDS).as('fields');
+});
+And('they select Resource subtype {string}', (subtype) => {
+  cy.get('[data-testid=publication-instance-type]').click({ force: true }).type(' ');
+  cy.get('@subTypes').then((subTypes) => {
+    cy.get(`[data-testid=${subTypes[subtype]}]`).click({ force: true });
+  });
+});
+And('they see a checkbox for "Is this a textbook?"', () => {
+  cy.get('[data-testid=is-textbook-checkbox]').should('be.visible');
+});
+// end common steps
+
+//   @274
+//   Scenario: Creator navigates to the Resource Type tab and selects Resource type "Contribution to journal"
+
+//   @453
+//   Scenario: Creator navigates to Resource Type tab
+Then('they see the field for Type', () => {
+  cy.get('[data-testid=publication-context-type]').should('be.visible');
+});
+And('they see the tab Description is clickable', () => {
+  cy.get('[data-testid=nav-tabpanel-description]').should('be.enabled');
+});
+And('they see the tab Resource Type is selected', () => {
+  cy.get('[data-testid=nav-tabpanel-description]').get('[tabindex=0]');
+});
+And('they see the tab Contributors is clickable', () => {
+  cy.get('[data-testid=nav-tabpanel-contributors]').should('be.visible');
+});
+And('they see the tab Files and License is clickable', () => {
+  cy.get('[data-testid=nav-tabpanel-files-and-license]').should('be.visible');
+});
+And('they see Previous is enabled', () => {
+  cy.get('[data-testid=button-previous-tab]').should('be.enabled');
+});
+And('they see Next is enabled', () => {
+  cy.get('[data-testid=button-next-tab]').should('be.enabled');
+});
+And('they see Save is enabled', () => {
+  cy.get('[data-testid=button-save-registration').should('be.enabled');
+});
+
+//   Scenario: Creator sees that fields are validated on Resource Type tab

```

Comment:
skal det være et nummer her?

Reply:
Det er ikke noe nummer for dette scenarioet i feature-fila